db/seeds.rb

```diff
@@ -12,34 +12,34 @@
-third_post = Post.create(author: third_user, Title: 'Unraveling the Secrets of the Ancient Ruins', Text: 'Walking amidst the ancient ruins, I felt a connection to the past, lost in time. Each crumbling stone whispered tales of a forgotten civilization waiting to be discovered.', CommentsCounter: 0, LikesCounter: 0, )
-fourth_post = Post.create(author: fourth_user, Title: 'Sunset Serenade by the Ocean', Text: 'As the sun dipped below the horizon, the ocean sang its evening symphony. The crashing waves and gentle breeze lulled me into a state of tranquility.', CommentsCounter: 0, LikesCounter: 0, )
-fifth_post = Post.create(author: fifth_user, Title: 'In the Footsteps of History: Exploring Ancient Cities', Text: 'Strolling through ancient cities, I marveled at the architectural wonders built centuries ago. The cobblestone streets echoed with the whispers of bygone civilizations.', CommentsCounter: 0, LikesCounter: 0, )
-sixth_post = Post.create(author: sixth_user, Title: 'Savoring Culinary Delights Around the World', Text: 'From the sizzling street food in bustling markets to the elegant haute cuisine in Michelin-starred restaurants, my taste buds embarked on a global culinary adventure."', CommentsCounter: 0, LikesCounter: 0, )
-seventh_post = Post.create(author: first_user, Title: 'A Night Under the Stars: Camping in the Wilderness', Text: 'Cocooned in a cozy tent, I gazed up at the twinkling constellations overhead. The crackling campfire and the distant howls of wildlife created an unforgettable night in the wilderness.', CommentsCounter: 0, LikesCounter: 0, )
-eighth_post = Post.create(author: second_user, Title: 'Diving into the Azure Depths: Exploring Coral Reefs', Text: 'Submerged in the vibrant underwater world, I swam alongside colorful coral reefs teeming with a kaleidoscope of marine life. The ocean\'s beauty left me mesmerized.', CommentsCounter: 0, LikesCounter: 0, )
-nineth_post = Post.create(author: third_user, Title: 'Lost in the Pages: A Book Lover\'s Retreat', Text: 'Immersed in the embrace of my favorite books, I found solace in a cozy nook surrounded by literary treasures. The written words transported me to far-off lands and fantastical realms.', CommentsCounter: 0, LikesCounter: 0, )
-tenth_post = Post.create(author: first_user, Title: 'Through the Lens: Capturing Nature\'s Beauty', Text: 'With camera in hand, I embarked on a photographic journey to capture the essence of nature\'s wonders. Each shot revealed the raw beauty of landscapes and wildlife.', CommentsCounter: 0, LikesCounter: 0, )
+first_post = Post.create(author: first_user, title: 'Exploring the Enchanting Forest', text: 'Venturing deep into the mystical woods, I discovered a hidden world of ancient trees and magical creatures. The air was filled with a sense of wonder as I wandered through the lush greenery.', comment_counter: 0, likes_counter: 0, )
```

Comment:
Good work implementing seeds to make your project easy to run and check. But kindly fix an error while trying to run seeds, so the next reviewer can review your project.

![image](https://github.com/Ouail-01/Blog-App/assets/93630700/a309406e-09b3-4c12-a4a8-555c80a22ef9)


I've been trying to create a user to avoid this error and isn't possible too. Please review your project before submitting it and make sure it's work correctly 👍🏼